content/en/serverless/installation/python.md

```diff
@@ -236,18 +159,54 @@ To install the Datadog Lambda Library with the CloudFormation macro, follow thes
 
 ### Subscribe the Datadog Forwarder to the Log Groups
 
-You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, to send metrics, traces and logs to Datadog.
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, to send metrics, traces, and logs to Datadog.
 
 1. [Install the Datadog Forwarder][2] if you haven't.
 2. [Ensure the option DdFetchLambdaTags is enabled][3].
 3. [Subscribe the Datadog Forwarder to your function's log groups][4].
 
 
-[1]: https://github.com/DataDog/datadog-lambda-layer-python/releases
+[1]: https://github.com/DataDog/datadog-lambda-python/releases
 [2]: https://docs.datadoghq.com/serverless/forwarder/
 [3]: https://docs.datadoghq.com/serverless/forwarder/#experimental-optional
 [4]: https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/#collecting-logs-from-cloudwatch-log-group
 {{% /tab %}}
+{{% tab "Datadog CLI" %}}
+
+<div class="alert alert-warning">This service is in public beta. If you have any feedback, contact <a href="/help">Datadog support</a>.</div>
+
+Use the Datadog CLI to set up instrumentation on your Lambda functions in your CI/CD pipelines. The CLI command automatically adds the Datadog Lambda library to your functions by using layers, and configures your functions to send metrics, traces, and logs to Datadog.
+
+### Install the Datadog CLI
+
+Install the Datadog CLI by using NPM or Yarn:
```

Comment:
Can we install with pip?

Reply:
nope, the CLI is written in Node.js instead of Python